app/models/hooks/web_hook.rb

```diff
@@ -44,7 +47,10 @@ def execute(data)
       }
       WebHook.post(post_url,
                    body: data.to_json,
-                   headers: {"Content-Type" => "application/json"},
+                   headers: {
+                    "Content-Type" => "application/json",
```

Comment:
Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.